go/vt/mysqlctl/mysqlshellbackupengine.go

```diff
@@ -0,0 +1,340 @@
+package mysqlctl
+
```

Comment:
This file and `go/vt/mysqlctl/mysqlshellbackupengine_test.go` need a licence header.